Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -77,6 +77,47 @@ final class UIViewControllerExtensionsTests: XCTestCase {
 		}
 		XCTAssertEqual(alertController.preferredAction, alertController.actions[preferredButtonIndex])
 	}
+    
+    func testAddChildViewController() {
+        let parentViewController = UIViewController()
+        let childViewController = UIViewController()
+
+        XCTAssert(parentViewController.childViewControllers.isEmpty == true)
+        XCTAssertNil(childViewController.parent)
+        
+        parentViewController.addChildViewController(childViewController, toContainerView: parentViewController.view)
+        
+        XCTAssert(parentViewController.childViewControllers == [childViewController])
+        XCTAssertNotNil(childViewController.parent)
+        XCTAssertEqual(childViewController.parent, parentViewController)
+    }
+    
+    func testRemoveChildViewController() {
+        let parentViewController = UIViewController()
+        let childViewController = UIViewController()
+        
+        parentViewController.addChildViewController(childViewController)
```

Comment:
You don't want to use the new `addChildViewController` method that you created?

Reply:
To be more clear: We do this with the implementation(we don't use a method in another method of the library) so why not with tests. This way we can safely remove one later.

Reply:
I guess that's a fair point. It's OK like this, then.